### PR TITLE
Adding Powered by Globus to the Footer

### PR DESCRIPTION
**Pull request by Sudershan29**
Screenshot 2024-07-15 at 4 50 33 PM

Screenshot 2024-07-15 at 4 51 28 PM

### CODE REVIEW

File: globus_portal_framework/templates/components/footer.html [hunk 1]
```diff
@@ -1,3 +1,11 @@
 <!-- Footer -->
+{% load static %}
 <footer>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark navbar-base-nav">
+        <div style="text-align: center; padding: 10px; display: flex; justify-content: center;">
+            <a href="https://www.globus.org" target="_blank">
+                <img src="{% static 'globus-portal-framework/images/powered-by-GO-2012.png'%}" alt="Powered by Globus.org" style="height: 50px">
+            </a>
+        </div>
+    </nav>
```
Comment (NickolausDS) [inline globus_portal_framework/templates/components/footer.html:10]: I think we could leave these changes off the older-style templates. That allows people to downgrade and keep the original look-and-feel of their site without also downgrading DGPF versions.

File: globus_portal_framework/templates/globus-portal-framework/v2/components/footer.html [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+{% load static %}
 <footer>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark navbar-base-nav">
+        <div style="text-align: center; padding: 10px; display: flex; justify-content: center;">
+            <a href="https://www.globus.org" target="_blank">
+                <img src="{% static 'globus-portal-framework/images/powered-by-GO-2012.png'%}" alt="Powered by Globus.org" style="height: 50px">
+            </a>
+        </div>
+    </nav>
 </footer>
```
Comment (NickolausDS) [inline globus_portal_framework/templates/globus-portal-framework/v2/components/footer.html:11]: Same for the v2 templates, probably better to remove this so older templates don't change unexpectedly.

File: globus_portal_framework/templates/components/footer.html (hunk 1 above)
Comment (Sudershan29) [reply globus_portal_framework/templates/components/footer.html]: I agree! Got them removed.

File: globus_portal_framework/templates/components/footer.html [hunk 3]
```diff
@@ -1,3 +1,4 @@
 <!-- Footer -->
+{% load static %}
```
Comment (NickolausDS) [inline globus_portal_framework/templates/components/footer.html:2]: I still see a the load static templatetags here

**Comment by NickolausDS** [comment]
The footer looks really good! I think it's a nice balance of adding attribution to Globus but also isn't obtrusive to viewing the page.

File: globus_portal_framework/templates/components/footer.html (hunk 3 above)
Comment (Sudershan29) [reply globus_portal_framework/templates/components/footer.html:2]: Got it removed 👍

**Review by NickolausDS — APPROVED** [review]
Looks good!

**Pull request merged**